src/main/cpp/JniHelpersCommon.h

```diff
@@ -63,23 +63,42 @@
 #define DEBUG _DEBUG
 #endif
 
+// Determines whether exceptions will be raised by JniHelpers
+#ifndef ENABLE_EXCEPTIONS
+#define ENABLE_EXCEPTIONS 1
+#endif
+
 // Determines whether logging messages will be printed by JniHelpers
 #ifndef ENABLE_LOGGING
 #define ENABLE_LOGGING 0
 #endif
 
+// Determines whether debug/info logging messages will be printed by JniHelpers
+#ifndef ENABLE_LOGGING_DEBUG
+#define ENABLE_LOGGING_DEBUG 0
```

Comment:
Given the name of this symbol, it should probably be defined to `= DEBUG` by default